### ISSUE TITLE
The idea

### ISSUE THREAD

**Issue by rayvafa** [tomdaniels/cafe-finder#1]
This is going to be a single page website. it will have some title and a form. 
The form contains following fields to start with:

1. latitude
2. longitude
3. radius
4. Search button

When the user click on search the page does not refresh. it will call Google Places API to fetch data for all the cafes inside that circle and it will render them like a list in the page.

Google Places API link: `https://developers.google.com/places/web-service/search`

 Notes:

- we will replace the lat,lng fields in the form with Geolocation API later. Then the app will read users geolocation from mobile/desktop machine automatically.
- We can sort the Cafes by their `review score`.
- We can add filter functionality later. e.g. searching for All cafes with more than 4 star rating

**Comment by tomdaniels** [comment]
@rayvafa have built the single web page. The geolocator is most prominent for ease of use though there is an option to search manually. (This is all a framework at this point). 
142588dae6a1584ab5bd175dfa19cf0699ca80ce
Would searching by lat & long really be the best way to go? Compared to searching by suburb for example?

**Comment by rayvafa** [comment]
yeah, I thought people might be interested to find cafes close to them, based on where they are and a close radius like 10 meters. I wouldn't be interested to know that in the other side of the suburb there is 5 star cafe! I wouldn't walk that far just for a covfefe! ;)

**Comment by tomdaniels** [comment]
I agree @rayvafa, though the first container would be the geolocator search. So the first thing they see is the cafe within 100m of them (automatically). With the option to scroll down & search an area they know they _will_ be in later and use a map they can drag around to see exactly where they need to go sort of thing. 

That's my train of thought, are there any limitations around having the map/drag feature like the one used on Domain as a 'coincidental example' 😉

**Comment by rayvafa** [comment]
that's right, I was thinking of sorting based on review score, rather than distance. 
map is a good idea. we can add that in the next iteration.
